auslib/web/admin/swagger/api.yaml

```diff
@@ -471,59 +370,20 @@ paths:
         Returns a JSON object containing metadata about Releases in Balrog's database. Due to its size, the actual Release 'blob' is never returned from this endpoint. There are a few query arguments that affect its response. If no arguments are provided, it returns all information about all of the Releases in the database. [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#releases)
       tags:
         - Releases
-      operationId: auslib.web.admin.views.mapper.release_get
+      operationId: auslib.web.common.releases.get_releases
       consumes: []
       produces:
         - application/json
       externalDocs:
         url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#releases"
         description: "Returns a JSON Object containing metadata about Releases in Balrog's database."
       parameters:
-        - $ref: '#/parameters/productParam'
-        - name: name_prefix
-          in: query
-          description: "If 'name_prefix' is passed, only Releases whose name starts with the given prefix will be returned."
-          type: string
-          x-nullable: true
-          required: false
-        - name: names_only
-          in: query
-          description: "If 'names_only' is set to true, the response changes format and provides a list of all of the Release names in the database"
-          type: string
-          x-nullable: true
-          required: false
-
+        - $ref: '#/parameters/productQuery'
+        - $ref: '#/parameters/namePrefixQuery'
+        - $ref: '#/parameters/namesOnlyQuery'
```

Comment:
This was to distinguish the path parameters from query parameters. Normally query parameters is optional. When we take a look on endpoints definitions, it is possible to determine the type of parameters that we are dealing.